src/models/glmnet.jl

```diff
@@ -70,9 +70,17 @@ function getfolds(resampling, X, y)
     return folds
 end
 
-function MLJBase.fit(model::GLMNetModel, verbosity::Int, X, y)
+function MLJBase.fit(model::GLMNetClassifier, verbosity::Int, X, y, weights)
```

Comment:
I think the correct interface it to only implement  `MLJBase.fit(model::GLMNetClassifier, verbosity::Int, X, y, weights=nothing)` with all positional arguments but weights defaults to nothing so that you don't have to have two function definitions. (see https://juliaai.github.io/MLJModelInterface.jl/stable/the_fit_method/)